### PR TITLE
Section2

### PR DESCRIPTION
**Pull request by NobodyKJ**
This is React Section2

### CODE REVIEW

File: sectionTwoCode/class15.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	constructer(){
+		this.gender = 'male';
+	}
+	
+	printGender(){
+		console.log(this.gender);
+	}
+}
+
+class Person extends Human {
```
Comment (LeonQC) [inline sectionTwoCode/class15.txt:40]: Try to review your course and get to know why use `extends` and how?

File: sectionTwoCode/class20.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//1, 2, 3
+//2, 4, 6
+
+const numbers = [1, 2, 3];
+const doubleNumArray = numbers.map((num) => {
```
Comment (LeonQC) [inline sectionTwoCode/class20.txt:5]: Get to know what is => ?

File: sectionTwoCode/class15.txt (hunk 1 above)
Comment (NobodyKJ) [reply sectionTwoCode/class15.txt:40]: Extending a Component allows us to pass props to a user defined class when a constructor is not present like it is in the App class. The extends keyword is used in class declarations or class expressions to create a class that is a child of another class.

File: sectionTwoCode/class20.txt (hunk 2 above)
Comment (NobodyKJ) [reply sectionTwoCode/class20.txt:5]: An arrow function expression is a compact alternative to a traditional function expression, but is limited and can't be used in all situations.

Differences & Limitations:

Does not have its own bindings to this or super, and should not be used as methods.
Does not have new.target keyword.
Not suitable for call, apply and bind methods, which generally rely on establishing a scope.
Can not be used as constructors.
Can not use yield, within its body.

**Pull request closed without merging**